configs/_base_/models/fcn_erfnet.py

```diff
@@ -0,0 +1,32 @@
+# model settings
```

Comment:
rename `configs/_base_/models/fcn_erfnet.py` to `configs/_base_/models/erfnet_fcn.py`

Reply:
I just followed name of [UNet](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/unet) which put decoder head (fcn, pspnet and deeplabv3) in front of backbone (unet). Altogether with `configs/_base_/models/fcn_unet_s5-d16.py`.